deployment/gcp/crawl.tmpl.yaml

```diff
@@ -40,7 +40,7 @@ spec:
         - name: CRAWL_DIRECTORY
           value: 'openwpm-crawl'
         - name: S3_BUCKET
-          value: 'openwpm-crawls'
+          value: 'openwpm-data'
```

Comment:
I was actually thinking we should have a team discussion about buckets and what they're for - I've been putting my crawl data in my team's bucket as I didn't want to pollute.

As this is a public repo, I don't think we should set a default here but prompt people to create / assign their s3 bucket.

Reply:
Yeah, that's seems reasonable and we should have that discussion.
But @englehardt just warned me about using the wrong bucket, so I thought we should make it safer for us.

Reply:
That's a fair point, but I agree with @vringar that it makes sense to swap to this for the time being.

Reply:
As for why Mozilla folks should use `openwpm-data` rather than `openwpm-crawls`: the former is in the same AWS region as our databricks instance.

Reply:
I created issue #44 to capture @birdsarah ' s input, but I can't quite figure out if this should just be another line in the README or an actual script that asks you some questions.